spec/hashie/mash_spec.rb

```diff
@@ -346,15 +346,25 @@ class SubMash < Hashie::Mash
     it 'responds to a set key with a suffix' do
       %w(= ? ! _).each do |suffix|
         expect(subject).to be_respond_to(:"abc#{suffix}")
-        expect(subject.method(:"abc#{suffix}")).to_not be_nil
       end
+
+      # it seems for ruby 2.2 :"abc#{suffix}" !== :"abc#{'='}" internally
```

Comment:
We know this is a bug, change this comment from "it seems ..." to "for ruby 2.2 ... + link to the bug".